framework/physics/physics_material/multi_group_xs/adjoint_mgxs.cc

```diff
@@ -22,17 +22,15 @@ AdjointMGXS::AdjointMGXS(const MultiGroupXS& xs) : xs_(xs)
     transposed_transfer_matrices_.push_back(S_ell_transpose);
   } // for ell
 
-  // transpose production matrices
+  // Transpose production matrices
   if (xs_.IsFissionable())
   {
+    transposed_production_matrices_.clear();
+    transposed_production_matrices_.resize(xs_.NumGroups());
     const auto& F = xs_.ProductionMatrix();
     for (size_t g = 0; g < xs_.NumGroups(); ++g)
-    {
-      std::vector<double> F_g_transpose;
       for (size_t gp = 0; gp < xs_.NumGroups(); ++gp)
-        F_g_transpose.emplace_back(F[gp][g]);
-      transposed_production_matrices_.push_back(F_g_transpose);
-    }
+        transposed_production_matrices_[g].push_back(F[gp][g]);
```

Comment:
I suppose we have an adjoint test case that has a fissile material? I am just worried about changes to the code that do not have a test case. If ti does not have one, can you add one. I can help verify the solution if need be, I like to do that sort of things!

Reply:
I do not believe we do. Could we table this on this PR? I have many changes coming to the adjoint capabilities in subsequent PRs. Since this has been in the code for awhile without a test, I would prefer to wait until the new adjoint capabilities are in for a test rather than making a test and changing it within a week or two.